src/main/java/dev/jbang/cli/Info.java

```diff
@@ -179,6 +184,10 @@ private void init(BuildContext ctx) {
 			}
 			gav = prj.getGav().orElse(null);
 			description = prj.getDescription().orElse(null);
+			docs = prj.getDocs().orElse(null);
+			if (prj.getMainSource() != null) {
```

Comment:
This `if` can be removed once we switch to `ResourceRef` (NB: any relative paths are relative to `Project.getResourceRef()`, not to the source files).

Reply:
Fixed by dcee2c8f3913144b4027efc8e94d84bd8e2d1ce1